ansible_ai_connect/main/tests/test_views.py

```diff
@@ -258,6 +259,7 @@ class TestChatbotView(TestCase):
 
     def setUp(self):
         super().setUp()
+        self.organization = Organization.objects.create(id=12345)
```

Comment:
We've a function called `create_user_with_provider()` in `ansible_ai_connect/test_utils.py` to provision test users.

And if you're test class depends on `WisdomServiceAPITestCaseBaseOIDC`, you get a test user created for Free! And the Teardown is also included :-)